<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name_line">
          <property name="text">
           <string>BlockName</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="description_line">
          <property name="text">
           <string>Whis will create a BLOCK that does something</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Icon Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="icon_line"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>.png</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <property name="topMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">background-color: none;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QRadioButton" name="facial_radio">
           <property name="text">
            <string>Facial</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QRadioButton" name="props_radio">
           <property name="text">
            <string>Props</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QRadioButton" name="clothes_radio">
           <property name="text">
            <string>Clothes</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QRadioButton" name="games_radio">
           <property name="text">
            <string>Games</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="presets_radio">
           <property name="text">
            <string>Presets</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="biped_radio">
           <property name="text">
            <string>Biped</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QRadioButton" name="other_radio">
           <property name="text">
            <string>Other</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="animals_radio">
           <property name="text">
            <string>Animals</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QRadioButton" name="data_radio">
           <property name="text">
            <string>Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Attrs</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="attr_name_1">
          <property name="text">
           <string>SetParent_string</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_1">
          <property name="text">
           <string>new_locator</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="attr_name_2">
          <property name="text">
           <string>Mirror_enum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_2">
          <property name="text">
           <string>True:False:Right_Only</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="attr_name_3">
          <property name="text">
           <string>TwistAmount_float</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_3">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="attr_name_4">
          <property name="text">
           <string>Ribbon_bool</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_4">
          <property name="text">
           <string>True</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="attr_name_5">
          <property name="text">
           <string>SetGameParent_string</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="attr_name_6">
          <property name="text">
           <string>Help_string</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_6">
          <property name="text">
           <string>Possible parent: *_Jnt </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLineEdit" name="attr_name_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="attr_name_8"/>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_8"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="attr_name_9"/>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_9"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="attr_name_10"/>
        </item>
        <item>
         <widget class="QLineEdit" name="attr_settings_10"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox">
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QRadioButton" name="simple_block">
           <property name="text">
            <string>Simple Block</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="sides_block">
           <property name="text">
            <string>L_ R_ Block</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="create_button">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
